.github/workflows/project.yml

```diff
@@ -34,7 +34,7 @@ jobs:
   integration-guardrails:
     strategy:
       matrix:
-        version: [12.0.0, 12, 14.0.0, 14, 16.0.0, 16, 18.0.0, 18.1.0, 20.0.0, 22.0.0]
```

Comment:
This was effectively testing that Node 12.0.0 is not crashing since the guardrails were incompatible with it. Now we test this in a different test suite that does `0.8` to `12` for that case. Actually, right now it does `0.8` to `10`, let me add `12.0.0` to it.

Reply:
Added to the other job.